nilearn/conftest.py

```diff
@@ -523,13 +535,13 @@ def f(img, part_name="right"):
 
 
 @pytest.fixture()
-def surface_glm_data(rng, surf_img):
+def surface_glm_data(rng, surf_img_2d):
     """Create a surface image and design matrix for testing."""
 
     def _make_surface_img_and_design(shape=5):
```

Comment:
```suggestion
    def _make_surface_img_and_design(n_samples=5):
```
I think this would be easier to understand

Reply:
yup
now made the changes locally